src/CidrType.php

```diff
@@ -27,7 +27,7 @@ public function convertToPHPValue(mixed $value, AbstractPlatform $platform): ?IP
         try {
             return IPBlock::create($value);
         } catch (InvalidArgumentException $e) {
-            throw ConversionException::conversionFailed($value, $this->getName(), $e);
+            throw new ConversionException('Could not convert database value "' . $value . '" to Doctrine Type ' . self::NAME, 0, $e);
```

Comment:
Not 100% sure but let's go with
ValueNotConvertible::new
https://github.com/doctrine/dbal/blob/4.0.x/src/Types/Exception/ValueNotConvertible.php#L22